riva/clients/asr/riva_asr_client.cc

```diff
@@ -258,26 +264,27 @@ class RecognizeClient {
   // Get a mutable reference to the Endpointing config message
   void UpdateEndpointingConfig(nr_asr::RecognitionConfig* config)
   {
-    if (!(start_history_ > 0 || start_threshold_ > 0 || stop_history_ > 0 || stop_history_eou_ > 0 || stop_threshold_ > 0)) {
-        return; 
+    if (!(start_history_ > 0 || start_threshold_ > 0 || stop_history_ > 0 ||
+          stop_history_eou_ > 0 || stop_threshold_ > 0)) {
```

Comment:
need to add stop_threshold_eou_ here, right?

Reply:
Updated with above commit.